platforms/hyperledger-fabric/configuration/roles/create/storageclass/templates/oke_sc.tpl

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ sc_name }}
+provisioner: blockvolume.csi.oraclecloud.com
+volumeBindingMode: WaitForFirstConsumer
+reclaimPolicy: Delete
```

Comment:
Please add newline at EOF

Reply:
Thanks sownak, I have made the requested update